zadanie_16

annbla7 opened the pull request.

annbla7 committed:
zadanie_16

kadyb commented on zadania/zadanie_16.py:
```diff
@@ -0,0 +1,25 @@
+
+def surface_distance(file):
+    points = []
+    with open(profil_csv, newline='', encoding='utf-8') as file:
```
Nazwa wejścia to `file` z funkcji, a nie `profil_csv`?

annbla7 replied:
Nazwa wejścia została poprawiona na profil_csv.

kadyb commented on zadania/zadanie_16.py:
```diff
@@ -0,0 +1,25 @@
+    for i in range(len(points) - 1):
+        x1, y1, z1 = points[i]
+        x2, y2, z2 = points[i + 1]
+        total_distance = math.sqrt((x2 - x1)**2 + (y2 - y1)**2 + (z2 - z1)**2)
```
Czy zwracanie całkowitej odległości działa? Chyba nigdzie nie jest akumulowana wartość?

annbla7 replied:
Wartości były nadpisywane zamiast dodawane. Zostało to poprawione.

annbla7 committed:
Poprawione zadanie_16.py

Zmiana nazwy wejścia z file na profil_csv oraz poprawienie w total_distance, aby wartości się dodawały, a nie nadpisywały.
